### ISSUE TITLE
Promisified (with util.promisify) `setTimeout` does not work

### ISSUE THREAD

**Issue by benjamingr** [sinonjs/fake-timers#223]
Node.js's setTimeout promisified [does](https://github.com/nodejs/node/blob/master/lib/timers.js#L459-L463):

```js
setTimeout[internalUtil.promisify.custom] = function(after, value) {
  return new Promise((resolve) => {
    active(new Timeout(resolve, after, [value], false));
  });
};
```

This means that it does not refer to setTimeout internally - we should override `setTimeout[internalUtil.promisify.custom]` if it is available to use our fake timers and revert it on uninstall.

If anyone wants to pick this up go ahead - otherwise I will work on a fix in ~2 weeks :)

**Comment by ehmicky** [comment]
I'm just bumping this, so the issue does not get stale.

**Comment by ehmicky** [comment]
Bumping this.

**Comment by fatso83** [comment]
We just need to add the `pinned` label, @ehmicky :-)

**Comment by mroderick** [comment]
This has been fixed in `@sinonjs/fake-timers@6.0.1`
